docs/REST API/rest-api.md

```diff
@@ -4569,16 +4567,29 @@ Get information about a bootstrap package that was uploaded to Fleet.
   "name": "bootstrap-package.pkg",
   "team_id": 0,
   "sha256": "6bebb4433322fd52837de9e4787de534b4089ac645b0692dfb74d000438da4a3",
-  "token": "AA598E2A-7952-46E3-B89D-526D45F7E233",
   "created_at": "2023-04-20T13:02:05Z"
 }
 ```
 
 In the response above:
-
-- `token` is the value you can use to [download a bootstrap package](#download-a-bootstrap-package)
 - `sha256` is the SHA256 digest of the bytes of the bootstrap package file.
 
+
+#### Example (download bootstrap package's contents)
```

Comment:
```suggestion
#### Example (download a bootstrap package)
```

Reply:
Was originally following the pattern from the [get script examples](https://fleetdm.com/docs/rest-api/rest-api#get-or-download-a-script) to keep the structure consistent, but I agree it makes more sense this way— I'll update those headings as well so they all match